wandb/sdk/wandb_run.py

```diff
@@ -1832,16 +1832,49 @@ def save(
         base_path: Optional[str] = None,
         policy: "PolicyName" = "live",
     ) -> Union[bool, List[str]]:
-        """Ensure all files matching `glob_str` are synced to wandb with the policy specified.
+        """Sync one or more files to W&B.
+
+        Relative paths are relative to the current working directory.
```

Comment:
shouldn't it be relative to the base_path?

Reply:
yikes, do we mentioned anywhere that we don't recurse?

Reply:
nope, I think it's a genuine bug!